examples/text_classification/tc_mnli_transformers.ipynb

```diff
@@ -0,0 +1,6283 @@
+{
```

Comment:
<p>same comment as before, there are too many logs, the notebook would look nicer without them</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/nlp/pull/434/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>